packages/react-strict-dom/src/native/modules/createStrictDOMComponent.js

```diff
@@ -545,6 +546,16 @@ export function createStrictDOMComponent<T, P: StrictProps>(
         style: { ..._styleProps.style }
       };
 
+      // Workaround: Android doesn't support ellipsis truncation if text is selectable
+      // See #136
+      if (
+        Platform.OS === 'android' &&
+        styleProps.numberOfLines != null &&
+        styleProps.style.userSelect === 'auto'
```

Comment:
can you clarify why we need to filter when `styleProps.style.userSelect === 'auto'`?

Reply:
If `userSelect` is already `none`, we don't need to do anything.